packages/core/src/extensions/Blocks/nodes/BlockTypes/BlockContentType.ts

```diff
@@ -0,0 +1,7 @@
+export type BlockContent<
```

Comment:
I think the following names make more sense:

`BlockType`

and then `HeadingType`, `HeadingBlockType`. Or maybe `HeadingBlock`. this one is more difficult

Reply:
I thought about this for a bit and I think what I like best is having a `BlockContainer` (currently `Block`) and `BlockContent` (currently also `BlockContent`, and `Heading`/`ListType`/`Paragraph` are types of `BlockContent`). I think they best describe what the nodes actually do and also best describe what we refer to as a block, as imo they're all parts of a block, not a full block themselves. Thoughts on this?